react-app/src/components/UserProfile/ProfileBox.js

```diff
@@ -58,11 +58,11 @@ function ProfileBox({label, content, userFile}) {
     <>
       <span> </span>
       <input
-      type='file'
-      onChange={userFileSubmit}
-      name='imageFile'
-      id='imageButton'
-      disabled
+        type='file'
+        onChange={userFileSubmit}
+        name='imageFile'
+        id='imageButton'
+        // disabled
```

Comment:
I had this disabled to keep it in theme with the rest of the page. I'll approve, but I think this should be reverted.